content/references/data-api.md

```diff
@@ -2874,41 +2844,43 @@ Retrieve validation votes, including votes for ledger versions that are outside
 ```
 
 <!-- MULTICODE_BLOCK_END -->
 
-[Try it! >](data-api-v2-tool.html#get-validations)
+This method requires the following URL parameters:
+
+| Field     | Value                           | Description           |
+|:----------|:--------------------------------|:----------------------|
+| `pubkey`  | String - Base-58 [Public Key][] | Validator public key. |
 
 Optionally, you can provide the following query parameters:
 
-| Field        | Value                  | Description                          |
-|:-------------|:-----------------------|:-------------------------------------|
-| `start`      | String - [Timestamp][] | Filter results to this time and later. |
-| `end`        | String - [Timestamp][] | Filter results to this time and earlier. |
-| `limit`      | Integer                | Maximum results per page. The default is 200. Cannot be more than 1000. |
-| `marker`     | String                 | [Pagination](#pagination) key from previously returned response. |
-| `format`     | String                 | Format of returned results: `csv` or `json`. The default is `json`. |
-| `descending` | Boolean                | If `true`, return results sorted with earliest first. Otherwise, return oldest results first. The default is `false`. |
+| Field    | Value   | Description                                             |
+|:---------|:--------|:--------------------------------------------------------|
+| `limit`  | Integer | Maximum results per page. Defaults to 200. Cannot be more than 1000. |
+| `marker` | String  | [Pagination](#pagination) key from previously returned response. |
+| `format` | String  | Format of returned results: `csv` or `json`. Defaults to `json`. |
 
 #### Response Format
 
 A successful response uses the HTTP code **200 OK** and has a JSON body with the following:
 
-| Field         | Value                           | Description                |
-|:--------------|:--------------------------------|:---------------------------|
-| `result`      | String                          | The value `success` indicates that the body represents a successful response. |
-| `count`       | Integer                         | Number of validation votes returned. |
-| `marker`      | String                          | (May be omitted) [Pagination](#pagination) marker. |
-| `validations` | Array of [Validation Objects][] | The requested validation votes. |
+| Field       | Value                         | Description                    |
+|:------------|:------------------------------|:-------------------------------|
+| `result`    | String                        | The value `success` indicates that the body represents a successful response. |
+| `count`     | Integer                       | Number of manifests returned.  |
+| `marker`    | String                        | (May be omitted) [Pagination](#pagination) marker. |
+| `manifests` | Array of [Manifest Objects][] | The requested manifests.       |
```

Comment:
The `[Manifest Objects][]` link is not showing up in generated HTML. 

It looks like a Manifest Objects section needs to be created?

Reply:
Good catch. Working on it.